### PR TITLE
Feature - ThongNguyen - TabbarController

### PR DESCRIPTION
**Pull request by mvn-thongnguyen-dn**
-----

### CODE REVIEW

File: 04_View/ThongNT/View/View/View/View/LoginViewController.swift [hunk 1]
```diff
@@ -72,6 +81,14 @@ final class LoginViewController: UIViewController {
         view.endEditing(true)
     }
     
+    @IBAction private func registerTouchUpInside(_ sender: UIButton) {
+        navigationController?.pushViewController(RegisterViewController(), animated: true)
+    }
+    
+    @IBAction private func ForgotPasswordTouchUpInside(_ sender: UIButton) {
```
Comment (mvn-bienle-dn) [inline 04_View/ThongNT/View/View/View/View/LoginViewController.swift:88]: Wrong name

**Review by mvn-bienle-dn — COMMENTED** [review]
@mvn-thongnguyen-dn Please check my comments. Thank you!

**Comment by mvn-thongnguyen-dn** [comment]
@mvn-bienle-dn Thanks for your comments!
Please check my fixes.

**Review by mvn-bienle-dn — APPROVED** [review]
LGTM!

**Pull request merged**